src/_playground/documentation/utils/getSourceModules.js

```diff
@@ -0,0 +1,9 @@
+const getSourceModules = (sourceModulePath) => {
+    const cleanSource = sourceModulePath.replace('./src', '');
+
+    const sourceModules = require('../../../../src' + cleanSource);
+
+    return sourceModules;
+};
+
+export default getSourceModules;
```

Comment:
Other plural references should probably be renamed too (in this file and in `Import` and `Properties`).